spec/cli/include_spec.lua

```diff
@@ -0,0 +1,28 @@
+local util = require("spec.util")
+
+describe("-I --include argument", function()
+   it("adds a directory to package.path", function()
+      local name = util.write_tmp_file(finally, "foo.tl", [[
+         require("add")
+         local x: number = add(1, 2)
+
+         assert(x == 3)
+      ]])
+
+      local pd = io.popen("./tl -I spec check " .. name, "r")
+      local output = pd:read("*a")
+      util.assert_popen_close(true, "exit", 0, pd:close())
+      assert.match("0 errors detected", output, 1, true)
+   end)
+   it("adds a directory to package.cpath", function()
+      local name = util.write_tmp_file(finally, "foo.lua", [[
+         print(package.cpath)
+      ]])
+      local pd = io.popen("LUA_CPATH=\"/usr/lib/lua/?.so\" ./tl run -I spec/cli/ " .. name, "r")
```

Comment:
I think this test will fail if people have things like LUA_CPATH_5_3 set in their environments (I do :grin: ). I believe it's okay to just do `print(package.cpath:match("spec/cli/") ~= nil)` above and check for `true` below (should also keep it crossplatform enough!)

Reply:
Good call! Done.